build-logic/buildquality/src/main/kotlin/gradlebuild/incubation/action/IncubatingApiReportAggregationParameter.kt

```diff
@@ -18,10 +18,13 @@ package gradlebuild.incubation.action
 
 import org.gradle.api.file.ConfigurableFileCollection
 import org.gradle.api.file.RegularFileProperty
+import org.gradle.api.provider.Property
 import org.gradle.workers.WorkParameters
 
 
 interface IncubatingApiReportAggregationParameter : WorkParameters {
     val reports: ConfigurableFileCollection
     val htmlReportFile: RegularFileProperty
+    val csvReportFile: RegularFileProperty
+    val currentCommit: Property<String>
```

Comment:
❓ Should it be a value source? It's a PITA for CC to depend on the commit

Reply:
As far as I understand this: since we use providers.exec it's not a cc input. So it should not invalidate configuration cache. 

@mlopatkin can you maybe confirm/deny my understanding?

Reply:
@asodja, value sources do not automatically make their content a build input. They can serve as regular providers that get resolved only after configuration time (e.g. at task input snapshotting time), given they are only plugged into the task inputs, which is the case here. For instance, Value sources back environment variable suppliers, which are definitely not automatically become a build input, etc.